README.md

```diff
@@ -315,3 +316,63 @@ Comp 40 gives you the tools to explore a large number of other topics:
+* Before coding, try to draw out what you would like to do. After drawing out your idea, go talk to a TA to see if your code makes logical sense. Even complex ideas like recursion will all make sense once you begin drawing them out. Spend Friday understanding the documentation from the homework assignment, spend Saturday drawing out your idea, spend Sunday talking with your TA about the sketch that you have and then code! The TAs prefer to talk about your sketch and idea than debug with you because debugging code is not only tiring, it takes a lot longer for you to finish your homework or project. Save the misery on both parts by trying out this strategy! If you have a good design, then you will avoid the mess of getting into bugs that are difficult to find. Moreover, if you have a good design, then the TA will likely not have to look at your code and fix your bug by noticing an error in your design. Trust me on this, you will save a SIGNIFICANT amount of time! 
+* Become acquainted with data structures. Learn how they work and draw them out with the TAs. Seek the TAs to learn how the complex data structures work instead of seeking them solely for debugging! The TAs for comp15 are an awesome group and all have a very deep understanding of how the data structures work. You will see that even one wrong detail in data structures will lead to segmentation fault quicksand or unexpected behavior. Each code you write really matter, so even if you understand, go check with the TAs to make sure you are crystal clear!
+* Decisions, decisions, decisions! There are multiple right answers in data structures, which is why, “am I doing it right?” will give you a very mixed answer in this class. Taking ownership of your decision after evaluating different options is a hallmark of a good engineer, so make sure you have a deep understanding of each data structure you cover in class. There is nothing minor that you can skip in this class, as each new data structure is now a new option you can consider for your upcoming homework or project. Each data structure has its own pros and cons, so learn how to evaluate and choose what you believe is the best for the problem you are trying to solve! 
```

Comment:
There are obvious benefits to being able to make a decision and take ownership of it- to a certain extent. You should be sure that the decisions you make for the data structures you choose to use are well-informed. I don't want people to see this and assume that they can pick whatever data structures they feel like, and just start writing code because they have "taken ownership" of it.